chapter_1/exercise_1_22/fold_line.c

```diff
@@ -13,11 +13,11 @@ int main(void)
   char line[MAXLINE];
   char fold_str[MAXLINE];
 
-  get_line(line, MAXLINE);
-
-  fold_line(line, fold_str, 30);
-
-  printf("%s", fold_str);
+  while ((get_line(line, MAXLINE)) > 0)
+  {
+    fold_line(line, fold_str, 40);

```

Comment:
@illustrofia Please don't use magic numbers in the code. Move the number 40 into a define, the performance will not be affected, but the readability will be significantly increased.